common/winobjc.nuproj.common.targets

```diff
@@ -1,6 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+  <Target Name="DisableVcxprojPack" BeforeTargets="BeforeResolveReferences" Condition="'$(SkipGetPackageContentsForOtherPlatforms)' != 'true' AND '$(Platform)' == 'AnyCPU'">
```

Comment:
Biiiiiigggg comments here explaining what and why and when and where are _absolutely appreciated_